Mathlib/Algebra/Order/Group/Cyclic.lean

```diff
@@ -0,0 +1,71 @@
+/-
+Copyright (c) 2025 María Inés de Frutos-Fernández, Filippo A. E. Nuccio. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: María Inés de Frutos-Fernández, Filippo A. E. Nuccio
+-/
+import Mathlib.GroupTheory.SpecificGroups.Cyclic
+
+/-!
+# Cyclic linearly ordered groups
+
+This file contains basic results about cyclic linearly ordered groups and their subgroups.
+
+The definitions `LinearOrderedCommGroup.Subgroup.gen_lt_one` (*resp.*
+`LinearOrderedCommGroup.gen_lt_one`) yields a generator of a non-trivial subgroup of a linearly
```

Comment:
And here

Reply:
fixed

Reply:
@Ruben-VandeVelde @erdOne Do you have more comments on this?

Reply:
@faenuccio can you resolve the conversations which you've dealt with, using the "Resolve conversation" button? Right now it looks like there are several things which you're still to deal with.